samples/sample-project/build.gradle.kts

```diff
@@ -81,7 +81,7 @@ dependencies {
     // repositories listed in the repository section
     // availLibrary("avail:example-lib:1.2.3")
     testImplementation(kotlin("test"))
-    implementation("avail:avail-core:+")
+    implementation("avail:avail:+")
```

Comment:
I'm not familiar with this syntax.  Does it just grab the most recent version?

Reply:
Yup. The sample project will get reworked after we publish and I update the Avail Gradle plugin project.